src/Utilities.UnitTests/ToolLocationHelper_Tests.cs

```diff
@@ -1957,15 +1957,15 @@ public void GetPathToReferenceAssembliesDefaultLocationNullFrameworkName()
         /// </summary>
         [Fact]
         [Trait("Category", "mono-osx-failing")]
-        public void GetPathToReferenceAssembliesDefaultLocation45()
+        public void GetPathToReferenceAssembliesDefaultLocation48()
```

Comment:
As I read it this should have been passing before because .NET 4.5 reference assemblies weren't installed. Is it right to update it? If so please update the comments above too.

Reply:
I believe the first method for finding "Version45" isn't particularly accurate. It looks for the appropriate folder where it should be, but that ends in "v4.0" rather than "v4.5" because of in-place replacements. Then, when it finds it, it returns successfully.

The method for finding all paths is more intelligent in that it actually finds version 4.5 if it exists. If not, it says Count is 0 and fails. Updating to 48 means it's actually there, so Count was 1, and this passed.

I might be misremembering some part of that. Honestly, I think we should really just unify these two paths, but they're rather confusing, and I don't want to commit to not messing that up.

Reply:
Please include this information in a comment or the commit message.